influxdb/influxdb08/dataframe_client.py

```diff
@@ -13,6 +13,16 @@
 from .client import InfluxDBClient
 
 
+def total_seconds(td):
```

Comment:
I do understand that this workaround is uglyfying the code for unsupported version of Python. Maybe we could put it into separate branch or I can leave it just in my fork.